src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -401,6 +402,47 @@ def get_command_cmn(daemon, cmd_str, op, st_idx, vals, bool_values):
         cmd_args.append(CommandArgument(daemon, cmd_enable, vals[idx]))
     return [cmd_str.format(*cmd_args, no = CommandArgument(daemon, cmd_enable))]
 
+def hdl_srv6_locator(daemon, cmd_str, op, st_idx, vals, bool_values):
```

Comment:
no unit tests at all?

Reply:
https://github.com/eddieruan-alibaba/sonic-mgmt/blob/eruan-testcasemaster/ansible/vars/configdb_jsons/7nodes_cisco/PE1.json#L8

in https://github.com/sonic-net/sonic-mgmt/pull/13785

Result could be found at https://aisportal.oss-cn-hangzhou.aliyuncs.com/phoenixwing/log/basesanity/basesanity_242_log.tar

Test results sneak view. 

srv6/test_srv6_basic_sanity.py::test_interface_on_each_node PASSED       [ 14%]
srv6/test_srv6_basic_sanity.py::test_check_bgp_neighbors PASSED          [ 28%]
srv6/test_srv6_basic_sanity.py::test_check_routes PASSED                 [ 42%]
srv6/test_srv6_basic_sanity.py::test_traffic_check PASSED                [ 57%]
srv6/test_srv6_basic_sanity.py::test_traffic_check_local_link_fail_case PASSED [ 71%]
srv6/test_srv6_basic_sanity.py::test_traffic_check_remote_igp_fail_case PASSED [ 85%]
srv6/test_srv6_basic_sanity.py::test_traffic_check_remote_bgp_fail_case PASSED [100%]